transformers/test_data/test_helpers.go

```diff
@@ -65,8 +66,13 @@ func CreateTestHeader(db *postgres.DB) int64 {
 // Create a header sync log to reference in an event, returning inserted header sync log
 func CreateTestLog(headerID int64, db *postgres.DB) core.EventLog {
```

Comment:
seems weird that this function would create only `tend` events - wonder if it's worth creating a new function if we need this to be a `tend` somewhere (and/or change it so we can pass in the `topic0`)

Reply:
Ah yeah.. this thing is called a bunch of other places. I should just set the topics after.